mteb/tasks/Classification/multilingual/MultilingualSentimentClassification.py

```diff
@@ -96,13 +96,13 @@ class MultilingualSentimentClassification(AbsTaskClassification, MultilingualTas
     def dataset_transform(self):
         # create a train set from the test set for Welsh language (cym)
         lang = "cym"
```

Comment:
Seem like this line might have been here for testing and needs to be removed as well

Reply:
This is not for testing, but `cym` have only train split in it. I can move definition of `lang` inside if to make it more clear

Reply:
Ahh I see. Just ran a test on this an it seems to work as expected - thanks for the fix